lib/babe/babe.go

```diff
@@ -33,11 +33,16 @@ import (
 	log "github.com/ChainSafe/log15"
 )
 
+// TODO: update #1233
 var (
 	// MaxThreshold is the maximum BABE threshold (node authorized to produce a block every slot)
 	MaxThreshold = big.NewInt(0).SetBytes([]byte{0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff, 0xff})
 	// MinThreshold is the minimum BABE threshold (node never authorized to produce a block)
 	MinThreshold = big.NewInt(0)
+	// DefaultThresholdNumerator is the default threshold numerator
+	DefaultThresholdNumerator = big.NewInt(10)
+	// DefaultThresholdDenominator is the default threshold denominator
+	DefaultThresholdDenominator = big.NewInt(1)
```

Comment:
you can delete this entire code block including `MaxThreshold` and `MinThreshold`, `MaxThreshold` is equal to 1/1 and `MinThreshold` is equal to 0/1